test/args_parse.py

```diff
@@ -23,6 +24,11 @@ def parse_common_options(datadir=None,
   parser.add_argument('--num_epochs', type=int, default=num_epochs)
   parser.add_argument('--num_workers', type=int, default=num_workers)
   parser.add_argument('--log_steps', type=int, default=log_steps)
+  parser.add_argument(
+      '--port',
```

Comment:
sorry I should be more clear in the last comment, but can we renamed it to `profiler_port`?

Reply:
Ah ok, done.